packages/manager/modules/overthebox/src/overthebox/migration/offers/overTheBox-migration-offers.controller.js

```diff
@@ -0,0 +1,103 @@
+import set from 'lodash/set';
+
+export default class OverTheBoxMigrationOffersCtrl {
+  /* @ngInject */
+  constructor(
+    $q,
+    $scope,
+    $translate,
+    TucToast,
+    OverTheBoxMigrationService,
+    OvhApiOverTheBox,
+  ) {
+    this.$q = $q;
+    this.$scope = $scope;
+    this.$translate = $translate;
+    this.TucToast = TucToast;
+    this.OverTheBoxMigrationService = OverTheBoxMigrationService;
+    this.OvhApiOverTheBox = OvhApiOverTheBox;
+  }
+
+  $onInit() {
+    this.offers = [];
+    this.loading = true;
+
+    this.$q.all([this.getCurrentOffer(), this.getOffers()]).finally(() => {
+      this.loading = false;
+    });
+  }
+
+  getCurrentOffer() {
+    return this.OvhApiOverTheBox.v6()
+      .get({
+        serviceName: this.serviceName,
+      })
```

Comment:
You need to use $http.get in service. This library "OvhApiOverTheBox" is deprecated.